### PR TITLE
Support Python 3.9

### PR DESCRIPTION
**Pull request by terencehonles**
It's that time again (#266)  :sweat_smile:

### CODE REVIEW

**Comment by terencehonles** [comment]
Looks like I need to update pyenv again too... I'll get to that later

**Comment by terencehonles** [comment]
Created PR to fix the docker image you are using randomknowledge/docker-pyenv-tox#5

**Review by candrikos — APPROVED** [review]
LGTM.

**Comment by candrikos** [comment]
Hey there, the PR looks fine. Let's wait for the testing image to be approved and then re-run the tests before merging.

**Comment by terencehonles** [comment]
> Hey there, the PR looks fine. Let's wait for the testing image to be approved and then re-run the tests before merging.

@candrikos it looks like there are some issues building the docker image as it's gotten a bit larger. I adjusted your circle CI script to update the installed pyenv and either use the Python version in the container or be able to build a newer version. I also made it forward compatible with the change we are looking at making in https://github.com/randomknowledge/docker-pyenv-tox/pull/6 / https://github.com/randomknowledge/docker-pyenv-tox/pull/7

**Comment by patrys** [comment]
Is official support the only reason to explicitly cap compatible Python versions at the package manager level? Capping on a minor version is only causing inconvenience to the community as we are forced to wait for this (purely cosmetic) change to be released. Could the "Python versions 3.10 and beyond are currently unsupported" message live in the README instead?

**Review by candrikos — DISMISSED** [review]

File: setup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def get_file_content(filename):
     license="GPLv2",
     dependency_links=[],
     setup_requires=[],
-    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,<3.9",
+    python_requires=">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,<3.10",
```
Comment (youtux) [inline setup.py:27]: Is there a reason why 3.10 and upper are excluded? Usually, newer versions of python don't break backward compatibility (Python 2 -> 3 being the exception).

**Comment by candrikos** [comment]
@terencehonles can you please add a dummy commit squash and force push to retrigger the `appveyor` testing service?
It seems to have stuck.

**Comment by terencehonles** [comment]
> @terencehonles can you please add a dummy commit squash and force push to retrigger the `appveyor` testing service?
> It seems to have stuck.

Done

File: setup.py (hunk 1 above)
Comment (terencehonles) [reply setup.py:27]: I'm not sure if this PR is really the right place to be asking this. We both agree though:

- https://github.com/transifex/transifex-client/pull/303#issuecomment-708437075
- https://github.com/transifex/transifex-client/pull/303#issuecomment-708477512

**Comment by terencehonles** [comment]
> @terencehonles can you please add a dummy commit squash and force push to retrigger the `appveyor` testing service?
> It seems to have stuck.

@candrikos I think this is actually the same issue as last time https://github.com/transifex/transifex-client/pull/266#issuecomment-565380187 and you may need to copy my branch to the transifex repository and create a new PR to merge it in.

**Comment by Diontsoumas** [comment]
Closed in favor of https://github.com/transifex/transifex-client/pull/306

**Pull request closed without merging**